client/blocks/product-purchase-features/product-purchase-features-list/index.jsx

```diff
@@ -230,9 +235,6 @@ export default connect(
 		};
 	},
 	{
-		recordBusinessOnboardingClick: partial(
```

Comment:
Cases like this are actually ok. The `partial` will only be called once since the `mapDispatchToProps` argument is a plain object and as such is static.